scripts/darwin.sh

```diff
@@ -65,23 +71,28 @@ for disk in $DISKS; do
     done
   fi
 
-  echo "raw: $raw_device"
-
   if [[ "$protected" == "Yes" ]]; then
-    echo "protected: True"
+    echo "isReadOnly: True"
   else
-    echo "protected: False"
+    echo "isReadOnly: False"
   fi
 
   if [[ "$device" == "/dev/disk0" ]] || \
      [[ ( "$removable" == "No" ) || ("$removable" == "Fixed") ]] || \
      [[ ( "$location" =~ "Internal" ) && ( "$removable" != "Yes" ) && ( "$removable" != "Removable" ) ]] || \
      echo "$mountpoints" | grep "^/$"
   then
-    echo "system: True"
+    echo "isSystem: True"
   else
-    echo "system: False"
+    echo "isSystem: False"
   fi
 
+  echo "isVirtual: null"
```

Comment:
We know for sure that the current scripts don't return any virtual drive, so I think we can safely put `false` here.

Reply:
With the new APFS encrypted volumes, this actually doesn't hold up anymore (disk1 is a synthesized / virtual of disk0), so I left it "null", as we can't say for sure atm